vendor.conf

```diff
@@ -15,32 +15,32 @@ github.com/docker/go d30aec9fd63c35133f8f79c3412ad91a3b08be06
 github.com/docker/go-connections 7beb39f0b969b075d1325fecb092faf27fd357b6
 github.com/docker/go-events 9461782956ad83b30282bf90e31fa6a70c255ba9
 github.com/docker/go-metrics d466d4f6fd960e01820085bd7e1a24426ee7ef18
-github.com/docker/go-units 9e638d38cf6977a37a8ea0078f3ee75a7cdb2dd1
+github.com/docker/go-units 47565b4f722fb6ceae66b95f853feed578a4a51c # v0.3.3
 github.com/docker/swarmkit edd5641391926a50bc5f7040e20b7efc05003c26
 github.com/flynn-archive/go-shlex 3f9db97f856818214da2e1057f8ad84803971cff
-github.com/ghodss/yaml 0ca9ea5df5451ffdf184b4428c902747c2c11cd7
-github.com/gogo/protobuf v1.0.0
-github.com/google/go-cmp v0.2.0
-github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
+github.com/ghodss/yaml 0ca9ea5df5451ffdf184b4428c902747c2c11cd7 # v1.0.0
+github.com/gogo/protobuf v1.1.1
+github.com/golang/glog 23def4e6c14b4da8ac2ed8007337bc5eb5007998
 github.com/golang/protobuf v1.1.0
-github.com/google/btree 316fb6d3f031ae8f4d457c6c5186b9e3ded70435
-github.com/google/gofuzz 44d81051d367757e1c7c6a5a86423ece9afcf63c
-github.com/googleapis/gnostic e4f56557df6250e1945ee6854f181ce4e1c2c646
-github.com/gorilla/context v1.1
-github.com/gorilla/mux v1.1
+github.com/google/btree e89373fe6b4a7413d7acd6da1725b83ef713e6e4
+github.com/google/go-cmp v0.2.0
+github.com/google/gofuzz 24818f796faf91cd76ec7bddd72458fbced7a6c1
+github.com/googleapis/gnostic 7c663266750e7d82587642f65e60bc4083f1f84e # v0.2.0
+github.com/gorilla/context v1.1.1
+github.com/gorilla/mux v1.6.2
 gotest.tools v2.1.0
```

Comment:
we should make this list alphabetical again 😅

Reply:
yeah, I think I'm gonna do a commit that reorders them :wink: